kotti_content_proxy/views.py

```diff
@@ -37,6 +41,12 @@ class ContentProxyAddForm(AddFormView):
     add = ContentProxy
     item_type = _(u"ContentProxy")
 
+    def save_success(self, appstruct):
+        proxied_id = appstruct.pop('proxied_id')
+        proxied_object = DBSession.query(Node).filter_by(id=proxied_id).one()
```

Comment:
This can be done without (importing) ``DBSession``: ``Node.query.filter(Node.id == proxied_id).one()``.